plugins/generator-1.19.4/forge-1.19.4/templates/overlay.java.ftl

```diff
@@ -82,35 +82,34 @@ package ${package}.client.screens;
             </#if>
 
             <#list data.getComponentsOfType("Image") as component>
-                <#assign x = component.x - 213>
-                <#assign y = component.y - 120>
                 <#if hasProcedure(component.displayCondition)>
                         if (<@procedureOBJToConditionCode component.displayCondition/>) {
                 </#if>
                     RenderSystem.setShaderTexture(0, new ResourceLocation("${modid}:textures/screens/${component.image}"));
-                    Minecraft.getInstance().gui.blit(event.getPoseStack(), posX + ${x}, posY + ${y}, 0, 0,
+					<@calculatePos anchor_point=component.anchorPoint.name() comp_x=component.x comp_y=component.y/>

```

Comment:
I would prefer if that macro would not set values of posX and posY, but would instead be used inline replacing `posX, posY` as that would make the generated code shorter and cleaner.

Example, this:

`Minecraft.getInstance().gui.blit(event.getPoseStack(), posX, posY, 0, 0,`

would become

`Minecraft.getInstance().gui.blit(event.getPoseStack(), <@calculatePosition component/>, 0, 0,`

Suggestion applies to both versions